example/satosa/templates/qr_code.html

```diff
@@ -16,12 +16,22 @@
                 </div>
                 <br>
                 <div id="content-qrcode" class="row">
-                   <span id="content-qrcode-payload" style="
-                      display: flex;
-                      flex-direction: column;
-                      justify-content: center;
-                      align-items: center;">
-                   </span>
+                   <div id = "content-qrcode-payload">
+                    <qr-code
+                        contents="{{ qrcode_text }}"
+                        module-color="{{ qrcode_color }}"
+                        position-ring-color="{{ qrcode_color }}"
+                        position-center-color="{{ qrcode_color }}",
+                        style="
+                            width: {{ qrcode_size }}px;
+                            height: {{ qrcode_size }}px;
+                            margin: 2em auto;
+                            background-color: #fff;
+                        "
+                        >
+                        <img class="icon-qr-code" src="/static/wallet-it/wallet-icon-blue.svg" slot="icon" />
```

Comment:
please use the static endpoint defined in the conf and not a constant like this /static/...

then: pass the static endpoint within the template context

Reply:
passed qr-code logo path in conf file